pywren/storage/s3_service.py

```diff
@@ -0,0 +1,87 @@
+import boto3
+import os
+import botocore
+import json
+
+
+class S3Service(object):
+    """
+    A wrap-up around S3 boto3 APIs.
+    """
+
+    def __init__(self, s3config):
+        self.s3_bucket = s3config['bucket']
+        self.session = botocore.session.get_session()
+        self.s3client = self.session.create_client('s3',
+                            config=botocore.client.Config(max_pool_connections=50))
+
+    def get_storage_location(self):
+        """
+        Get storage location for this S3 service.
+        :return: S3 bucket name
+        """
+        return self.s3_bucket
+
+    def put_object(self, key, data):
+        """
+        Put an object in S3.
+        :param key: key of the object.
+        :param data: data of the object
+        :return: None
+        """
+        self.s3client.put_object(Bucket=self.s3_bucket, Key=key, Body=data)
+
+    def get_object(self, key):
+        """
+        Get object from S3 with a key.
+        :param key: key of the object
+        :return: Data of the object
```

Comment:
Can we specify the type of the return object ? i.e. is it a byte array or an iterator etc ?